lib/rules-generator.js

```diff
@@ -405,6 +463,9 @@ util.methods(Generator, {
         if (replaceExp()) {
```

Comment:
I go out of my way not to copy everything - maybe not worth the complexity of the code...